src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -735,11 +738,11 @@ private void InitializeWorkers(string language)
             };
 
             var providers = new List<IWorkerProvider>();
-            if (!string.IsNullOrEmpty(language))
+            if (!string.IsNullOrEmpty(_language))
             {
-                _startupLogger.LogInformation($"{ScriptConstants.FunctionWorkerRuntimeSettingName} is specified, only {language} will be enabled");
+                _startupLogger.LogInformation($"{ScriptConstants.FunctionWorkerRuntimeSettingName} is specified, only {_language} will be enabled");
                 // TODO: We still have some hard coded languages, so we need to handle them. Remove this switch once we've moved away from that.
-                switch (language.ToLower())
+                switch (_language.ToLower())
```

Comment:
I missed this earlier. Can you change this to use ToLowerInvariant()

Reply:
Will do